lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -240,36 +245,45 @@ Boto3 provides lots of useful information when an exception is thrown so pass th
 along with the message.
 
 ```python
-# Import ClientError from botocore
+from ansible.module_utils.ec2 import HAS_BOTO3
```

Comment:
Since some of these examples are using AnsibleAWSModule, can you add something about not needing to import or check for HAS_BOTO3 if using it?

Reply:
Done, thanks